src/lib/modules/Relations/backoffice/components/RelationCard/SeriesCard.js

```diff
@@ -18,7 +21,14 @@ export default class SeriesCard extends Component {
           {actions}
           {data.metadata.series_type || data.metadata.mode_of_issuance}
         </Card.Meta>
-        <SeriesIcon size="huge" color="grey" />
+        {recordToPidType(data) === 'serid' ? (
```

Comment:
Suggestion: variables!

Honestly it is pretty readable already but i just think it is always a good practice to save conditions into variables, they allow you to actually describe what you are evaluating!  😃 

```js
// or some other name if this is not the thing you are evaluating
const isSerialRecord = recordToPidType(data) === 'serid' 

isSerialRecord? <> : <>

```

_also tell me how you make the screen captures_

Reply:
generally I agree with Nicolases suggestion.
I would go step further though and try to understand why you need to check if the record is series? It is already specific `SeriesCard` so it should only be used for series, what was the need for this additional check?